mbed/mbed.py

```diff
@@ -1447,6 +1440,22 @@ def ignore_build_dir(self):
             except IOError:
                 error("Unable to write build ignore file in \"%s\"" % os.path.join(build_path, '.mbedignore'), 1)
 
+    def detect_target(self, info=None):
```

Comment:
What's the use case for detecting multiple targets? Also what if I have multiple targets of the same kind mixed with different ones?

I guess I'm missing the point.